backend-api/src/functions/asyncIssueBiometricCredential/asyncIssueBiometricCredentialHandler.test.ts

```diff
@@ -9,13 +9,20 @@ import {
   mockBiometricSessionId,
   mockSessionId,
 } from "../testUtils/unitTestData";
+import { successResult, emptyFailure } from "../utils/result";
 
 describe("Async Issue Biometric Credential", () => {
   let dependencies: IssueBiometricCredentialDependencies;
   let context: Context;
   let consoleInfoSpy: jest.SpyInstance;
   let consoleErrorSpy: jest.SpyInstance;
 
+  const mockGetSecretsSuccess = jest.fn().mockResolvedValue(
+    successResult({
+      mock_biometric_viewer_access_key: "mock_viewer_key",
```

Comment:
nitpick: typically we always use camel case unless we expect other cases, so I think your key casing is fine but we could change the value to be camel. Entirely non blocking

```suggestion
      mock_biometric_viewer_access_key: "mockViewerKey",
```